mrp_bom_sale_pack/README.rst

```diff
@@ -0,0 +1,20 @@
+Mrp Bom Sale Pack
+=================
+
+This module allows you to on the sale order, if a product has a 'set' type
```

Comment:
"This module allows shows on sales orders the list of components for products with bill of materials of type 'set'."